distribution/lib/Standard/Base/0.0.0-dev/docs/api/Data/XML/XML_Format.md

```diff
@@ -8,4 +8,4 @@
     - read self file:Standard.Base.System.File.File on_problems:Standard.Base.Errors.Problem_Behavior.Problem_Behavior -> Standard.Base.Any.Any
     - read_stream self stream:Standard.Base.System.Input_Stream.Input_Stream metadata:Standard.Base.System.File_Format_Metadata.File_Format_Metadata= -> Standard.Base.Any.Any
     - resolve constructor:Standard.Base.Any.Any -> Standard.Base.Any.Any
-- Standard.Base.System.File_Format.File_Format_SPI.from that:Standard.Base.Data.XML.XML_Format.XML_Format -> Standard.Base.System.File_Format.File_Format_SPI
+- Standard.Base.System.File_Format.File_Format_SPI.from _:Standard.Base.Data.XML.XML_Format.XML_Format -> Standard.Base.System.File_Format.File_Format_SPI
```

Comment:
Signature for `File_Format_SPI.from (_:XML_Format) = File_Format_SPI.new XML_Format` (in https://github.com/enso-org/enso/blob/7f26fdeb1ea58639d4d9714b28b70e7d361bf697/distribution/lib/Standard/Base/0.0.0-dev/src/Data/XML/XML_Format.enso#L92)